### PR TITLE
Replace Spectrum with GitHub on blog posts

### PR DESCRIPTION
**Pull request by lfades**
Format of some code changed by pre-commit.

### CODE REVIEW

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**/blog/next-5-1**](https://next-site-git-use-discussions.zeit.now.sh/blog/next-5-1) | [**/blog/next-6-1**](https://next-site-git-use-discussions.zeit.now.sh/blog/next-6-1) |
|:-:|:-:|
|<a href="https://next-site-git-use-discussions.zeit.now.sh/blog/next-5-1">Screenshot of /blog/next-5-1</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink-cdn.s3.amazonaws.com/wXHjJYKngVToI77m5b4AWPM5_og_.png">(view full size)</a> |<a href="https://next-site-git-use-discussions.zeit.now.sh/blog/next-6-1">Screenshot of /blog/next-6-1</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink-cdn.s3.amazonaws.com/VdA__eJtIWzX_Tz8Hnx0rVEtygk_.png">(view full size)</a> |



| [**/blog/next-7**](https://next-site-git-use-discussions.zeit.now.sh/blog/next-7) | [**/blog/next-8-0-4**](https://next-site-git-use-discussions.zeit.now.sh/blog/next-8-0-4) |
|:-:|:-:|
|<a href="https://next-site-git-use-discussions.zeit.now.sh/blog/next-7">Screenshot of /blog/next-7</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink-cdn.s3.amazonaws.com/kKINR0I7Gcbi9kUfutxHYHDe4Uw_.png">(view full size)</a> |<a href="https://next-site-git-use-discussions.zeit.now.sh/blog/next-8-0-4">Screenshot of /blog/next-8-0-4</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink-cdn.s3.amazonaws.com/VQp2QodSieJRitP__5QpBbvQSeo_.png">(view full size)</a> |



| [**/blog/next-8**](https://next-site-git-use-discussions.zeit.now.sh/blog/next-8) | [**/blog/next-9-0-7**](https://next-site-git-use-discussions.zeit.now.sh/blog/next-9-0-7) |
|:-:|:-:|
|<a href="https://next-site-git-use-discussions.zeit.now.sh/blog/next-8">Screenshot of /blog/next-8</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink-cdn.s3.amazonaws.com/hvaXdgDS5QmMfKl3uELydYILIwU_.png">(view full size)</a> |<a href="https://next-site-git-use-discussions.zeit.now.sh/blog/next-9-0-7">Screenshot of /blog/next-9-0-7</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink-cdn.s3.amazonaws.com/u0mUPLbLrmo3lUt7iJogwQeTTQs_.png">(view full size)</a> |



And 4 other routes:
- [**/blog/next-9-1-7**](https://next-site-git-use-discussions.zeit.now.sh/blog/next-9-1-7)
- [**/blog/next-9-1**](https://next-site-git-use-discussions.zeit.now.sh/blog/next-9-1)
- [**/blog/next-9-2**](https://next-site-git-use-discussions.zeit.now.sh/blog/next-9-2)
- [**/blog/next-9**](https://next-site-git-use-discussions.zeit.now.sh/blog/next-9)



Commit bc56235014d9140a4d6bbb14a2db72b8ef6e9686 (https://next-site-git-use-discussions.zeit.now.sh).

File: pages/blog/next-6-1.mdx [hunk 1]
```diff
@@ -117,22 +119,32 @@ The result of this effort is [next&#8209;codemod](https://github.com/zeit/next-c
...
-- Checking the [good first issue](https://github.com/zeit/next.js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22) and [help wanted](https://github.com/zeit/next.js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22) labels on [GitHub](https://github.com/zeit/next.js)
-
-<Image src={'https://assets.zeit.co/image/upload/front' + '/blog/next6-1/good-first-issue.png'} width={1968/3} height={1468/3} />
-<Caption>There's 30 open issues with the <GenericLink href="https://github.com/zeit/next.js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22">good first issue</GenericLink> label. Giving new contributors the opportunity to contribute.</Caption>
+- Checking the [good first issue](https://github.com/zeit/next.js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22) and [help wanted](https://github.com/zeit/next.js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22) labels on [GitHub](https://github.com/zeit/next.js)<Image src={'https://assets.zeit.co/image/upload/front' + '/blog/next6-1/good-first-issue.png'} width={1968/3} height={1468/3} />
```
Comment (chibicode) [inline pages/blog/next-6-1.mdx]: `<Image src={'https://assets.zeit.co/image/upload/front' + '/blog/next6-1/good-first-issue.png'} width={1968/3} height={1468/3} />` is moved here

File: pages/blog/next-6-1.mdx [hunk 2]
```diff
@@ -63,10 +63,12 @@ One of these changes is that the `server/build` directory is moved to the top le
 
 We have also been focusing on adding [Flow](https://flow.org/) types throughout the codebase.
 
-A change we've made that is more visible to users is that `.next/dist` has been renamed to `.next/server`. The `.next` directory that holds build output. For example when you run `next build` the result will be stored in `.next`.
+A change we've made that is more visible to users is that `.next/dist` has been renamed to `.next/server`. The `.next` directory that holds build output. For example when you run `next build` the result will be stored in `.next`.<Image src={'https://assets.zeit.co/image/upload/front' + '/blog/next6-1/server-dist.png'} width={1574/3} height={754/3} />
```
Comment (chibicode) [inline pages/blog/next-6-1.mdx]: `<Image src={'https://assets.zeit.co/image/upload/front' + '/blog/next6-1/server-dist.png'} width={1574/3} height={754/3} />` is moved here

File: pages/blog/next-9-0-7.mdx [hunk 3]
```diff
@@ -248,6 +248,6 @@ We're excited to see the continued growth of the Next.js community.
 - We've had over **800** contributors landing at least 1 commit.
 - On GitHub, the project has been starred over **41,100** times.
 
-The Next.js community on spectrum.chat/next-js has doubled since the last major release with over 10,900 members. Join us!
```
Comment (timneutkens) [inline pages/blog/next-9-0-7.mdx]: We should keep these but remove the `on spectrum.chat/next-js`

File: pages/blog/next-8-0-4.mdx [hunk 4]
```diff
@@ -196,19 +196,11 @@ If you are looking to start contributing to Next.js, issues with the [good first
...
 
-The Spectrum community is a place where you can chat about Next.js, get advice on how to solve problems, and help out other community members with your knowledge of Next.js.
+GitHub discussions is a place where you can chat about Next.js, get advice on how to solve problems, and help out other community members with your knowledge of Next.js.
 
-[Join the community on Spectrum!](https://spectrum.chat/next-js)
-
-<Image
-  caption={<>The Spectrum community</>}
-  captionSpacing={0}
-  src="/static/blog/next-8-0-4/spectrum.png"
```
Comment (chibicode) [inline pages/blog/next-8-0-4.mdx]: We can probably remove this file

**Comment by timneutkens** [comment]
Great work Luis 💯

**Pull request merged**